### ISSUE TITLE
0.6 - ContinueOnException has issue?

### ISSUE THREAD

**Issue by mailingfeng** [sizovs/PipelinR#20]
always throw AggregateException?

```
public class ContinueOnException implements NotificationHandlingStrategy {

  @Override
  public void handle(List<Runnable> runnableNotifications) {
    Collection<Throwable> exceptions = new ArrayList<>();
    runnableNotifications.forEach(
        it -> {
          try {
            it.run();
          } catch (Throwable e) {
            exceptions.add(e);
          }
        });
    throw new AggregateException(exceptions);
  }
}
```

**Comment by sizovs** [comment]
Thanks for reporting. Fixed and released in 0.7.

**Comment by mailingfeng** [comment]
good job 🍺
